django-stubs/db/models/base.pyi

```diff
@@ -71,13 +71,15 @@ class Model(metaclass=ModelBase):
     def get_constraints(self) -> list[tuple[type[Model], Sequence[BaseConstraint]]]: ...
     def save(
         self,
+        *args: NoReturn,
```

Comment:
Sorry, I for some reason accepted this without asking, but now I got really interested: why not just `*`?

Reply:
That fails stubtest right now, since Django allows positional arguments still. Maybe that would be better for the types though, since users will probably want to adopt the new keyword-only format now?

Reply:
Yeah, I think that we should add an allow rule for that. Can you please?